FinancialTracker/AppDelegate.swift

```diff
@@ -13,9 +13,10 @@ import UserNotificationsUI
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterDelegate {
     
+    var authMananger: AuthManager?
```

Comment:
It should be optional because when I initialize authManager the Firebase class FirebaseAuth and its variably auth is called and if i have initalized this auth before:
```swift 
FirebaseApp.configure()
```
which is called in `didFinishLaunchingWithOptions` the program crashes.

Reply:
You can set it as `var authMananger: AuthManager!`. In this way you declare that it is always needed and it will not be used until it is configured, so no worries about crashing.